### PR TITLE
Added my participants YAML.

### PR DESCRIPTION
**Pull request by jeid64**
Hope this is correct.

### CODE REVIEW

File: people/2015/spring/jeid64.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+blog: http://fossclassjeid64.blogspot.com/
+feed: http://fossclassjeid64.blogspot.com/feeds/posts/default?alt=rss
+forges:
+    - http://github.com/jeid64
+irc: jeid64
+name: Julien Eid
```
Comment (decause) [inline people/2015/spring/jeid64.yaml]: Remove this field and I'll merge

**Comment by decause** [comment]
see above comment

Fix and I'll merge :wrench:

**Comment by jeid64** [comment]
Fixed! Ready to be merged.

**Comment by decause** [comment]
:tea:

**Pull request merged**